File: app/src/main/java/com/neome/api/meta/base/dto/DefnStudioMapOfTableHeader.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.dto.DefnDtoTableHeader;

import java.util.Map;

import com.neome.api.meta.base.Types.MetaIdHeader;

@SuppressWarnings("unused")
public class DefnStudioMapOfTableHeader
{
  public MetaIdHeader[] keys;

  public Map<MetaIdHeader, DefnDtoTableHeader> map;
}

File: app/src/main/java/com/neome/api/meta/base/dto/DefnStudioMapOfTableHeader.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.dto.DefnDtoTableHeader
import com.neome.api.meta.base.Types.MetaIdHeader

interface DefnStudioMapOfTableHeader
{
  val keys: Array<MetaIdHeader>
  val map: Map<MetaIdHeader, DefnDtoTableHeader>
}